pkg/utils/exec/cmd.go

```diff
@@ -72,8 +72,7 @@ func ForkExec(ctx context.Context, dir string, name string, arg ...string) error
 		return fmt.Errorf("write cmdline file %s: %w", cmdlinePath, err)
 	}
 
-	subCtx := context.Background()
```

Comment:
This context is intentionally new, to avoid the cancellation of forked processes on binary runtime